### ISSUE TITLE
Broken CI

### ISSUE THREAD

**Issue by panicbit** [slog-rs/slog#294]
It looks like CI doesn't run anymore.
I don't see travis CI showing [in this PR for example ](https://github.com/slog-rs/slog/pull/290).
Furthermore tests are failing with `--all-features`.

**Comment by Techcable** [comment]
Yeah, `travis-ci.org` has [shutdown](https://blog.travis-ci.com/2021-05-07-orgshutdown/). I *think* the new travis-ci.com is still free, but I believe most of the community has moved to Github Actions.

I'm going to switch over to using Github Actions.

**Comment by panicbit** [comment]
Yes, it's still free, but now you get a set amount of free credits that you have to keep re-requesting. I wonder why people moved to GitHub Actions 🤔 🤔 🤔 .

**Comment by nabijaczleweli** [comment]
.com is "free" insofar as you get 10000(?) "open-source credits". Then you pay. GHA is a good choice here.

**Comment by panicbit** [comment]
https://docs.travis-ci.com/user/billing-overview/#free-trial-plan

https://docs.travis-ci.com/user/billing-faq/#what-if-i-am-building-open-source

**Comment by Techcable** [comment]
Holly guacamole that's terrible 😮 

Sound like we need to switch to Github actions ASAP. @dpc gave my comment a heart so I think this counts as a go-ahead 😉

**Comment by dpc** [comment]
Go ahead! I want all my projects off Travis, just I don't have time to take care of it - it was great while it lasted, but can't blame them from running out of $ to subsidize this and many other projects. :)
